src/FTPObject.jl

```diff
@@ -104,7 +104,9 @@ Upload IO object "file" to the FTP server and save as "file_name".
 If "file" is not specified, the file "file_name" is uploaded.
 """ ->
 function upload(ftp::FTP, file_name::AbstractString, file=nothing)
```

Comment:
This is a very strange function signature. Maybe it should be broken up into two functions?
```julia
upload(ftp::FTP, file_name::AbstractString)
upload(ftp::FTP, file::IO)
```